designs/extensible-cli-and-scaffolding-plugins-phase-1.md

```diff
@@ -25,7 +25,8 @@ Each plugin would minimally be required to implement the `Plugin` interface.
 
 ```go
 type Plugin interface {
-    // Version returns the plugin's semantic version, ex. "v1.2.3".
+    // Version returns the plugin's version. This version, while being a
```

Comment:
While it is nice to have an up-to-date design doc to reference, it is perhaps prudent to have a separate doc that details the current plugin spec. This would be beneficial for when phase 2 is implemented.

Reply:
I agree. It could be in the Reference section in the book/docs. But it shows out of the scope of this PR. We can track a new issue to do just it.